core/application/impl/chain_spec_impl.cpp

```diff
@@ -101,10 +105,19 @@ namespace kagome::application {
     auto fork_blocks_opt = tree.get_child_optional("forkBlocks");
     if (fork_blocks_opt.has_value()
         && fork_blocks_opt.value().get<std::string>("") != "null") {
+      /*
+       * Currently there is no special handling implemented for forkBlocks.
+       * We read them, but do it in terms of legacy chain specs' support.
+       * We assume we are safe even if there are some values specified.
+       *
+       * The kagome node currently is going to sync with the main fork only.
+       * The full implementation of forkBlocks support is not worth it now but
+       * can be added later on.
+       */
+      log_->warn(
+          "A non-empty set of 'forkBlocks' encountered! They might not be "
```

Comment:
I think it might be better to include a part of the explanation from the comment into the warning text.